### DISCUSSION TITLE
Release v1.6.7 / 5.61.7

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2628] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is a maintenance release, it introduces minor changes and fixes a few issues.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added option to the classic ui to apply a supporter certificate

### Changed
- time limited certificates now have 1 extra free month of validity, to improve the renewal expirience

### Fixed
- fixed issue with Hebrew language (Classic UI) [#2608](https://github.com/sandboxie-plus/Sandboxie/issues/2608)
- fixed issue with startmenu integration and snapshots

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.7'>Release v1.6.7 / 5.61.7</a>.</em>

**Comment by Jeppa** [comment]
>added option to the classic ui to apply a supporter certificate

Does that mean the classic version has limitations now, too? (without cert)

**Reply by DavidXanatos** [reply]
You can use plus features in classic, by editing the ini file, to simplify the process of adding a cert to use these plus features these functions were added.

**Reply by Jeppa** [reply]
Alles klar ;)
